HelpSource/Classes/PV_ChainUGen.schelp

```diff
@@ -64,14 +147,11 @@ Examples::
 
 subsection:: pvcalc
 code::
-(
-s.waitForBoot({
-	c = Buffer.read(s, Platform.resourceDir +/+ "sounds/a11wlk01.wav");
-})
-)
+// a sound file
+c.free; c = Buffer.read(s, Platform.resourceDir +/+ "sounds/a11wlk01.wav");
```

Comment:
this could also be on multiple lines

Reply:
I'd like to keep it without parentheses as a single line in this case.